### DISCUSSION TITLE
Error - NameError Mdtraj

### DISCUSSION THREAD

**Discussion by pennysr** [schrodinger/pymol-open-source#348] [General]
I am not a linux guy, I know a reasonable amount but when it gets really in depth, I kind of get lost. I am having to help someone running Pymol and they are getting this error message:

NameError: free variable 'mdtraj' referenced before assignment in enclosing scope. 

I have zero ideas, not sure if its a config error or something they had typed or even a bug. I see lots of errors like this, not mdtraj but other modules where its reported as a bug or its what was entered by the person running it.

Any ideas?

**Comment by speleo3** [comment]
PyMOL doesn't use `mdtraj`. Such an error most likely comes from a [third-party plugin](https://pymolwiki.org/index.php/Plugins) or from custom code in `~/.pymolrc`.

Try running `pymol -k` to disable all plugins and custom code.

Check https://pymolwiki.org/index.php/pymolrc for broken code.

Check `~/.pymol/startup` for plugins.
